string  option               
string  group_name               
---
bool success     